src/scripts/lexmatch-sssom-compare.py

```diff
@@ -125,13 +125,15 @@ def extract_unmapped_matches(matches: TextIO, output_dir: str, summary: TextIO):
     condition_4 = msdf_mondo.df["object_id"].str.contains("Orphanet")
     condition_5 = msdf_mondo.df["object_id"].str.contains("DOID")
     condition_6 = msdf_mondo.df["object_id"].str.contains("NCIT")
+    condition_6A = msdf_mondo.df["object_id"].str.contains("GARD")
```

Comment:
The logic in this section, lines ~120 to ~300 look repetitive, but I'm not familiar enough to suggest a refactor. Maybe this will be refactored when that other refactor (the one where the `makefile` iterates through `ALL_COMPONENT_IDS` and passes them to lexmatch) is added.

Reply:
I am refactoring all of this in the latest PR.